### ISSUE TITLE
nx migrate should be suggested/written to all new workspace types, no ng update

### ISSUE THREAD

**Issue by JamesHenry** [nrwl/nx#4112]
## Current Behavior

For new Angular workspaces, within **packages/workspace/src/schematics/workspace/files/package.json__tmpl__**

```
    <% if(cli === 'angular') { %>
    "update": "ng update @nrwl/workspace",
    <% } %>

   <% if(cli === 'nx') { %>
    "update": "nx migrate latest",
    <% } %>
```

Therefore in the package.json the user has the `"update"` script pointing to `ng update`.

## Expected Behavior

The `"update"` script in the package.json should point to `nx migrate`.

## Steps to Reproduce

This issue may not be prioritized if details are not provided to help us reproduce the issue.

**Comment by vsavkin** [comment]
Fixed it here 06e13dae2fecb621d03a197b65d4e37c4c73e06d, will be part of v11. Thanks, James :)

**Comment by JamesHenry** [comment]
Thanks @vsavkin!

Intrigued by that major rename 👀
